ferrocene/ci/scripts/run-self-test.sh

```diff
@@ -10,9 +10,19 @@ COMMIT="$(git rev-parse HEAD)"
 BUCKET="ferrocene-ci-artifacts"
 PREFIX="ferrocene/dist/${COMMIT}"
 
-root="$(mktemp -d)"
+TAR="tar"
+FERROCENE_SELF_TEST="ferrocene-self-test"
+# Ensure we use GNU tar on Windows, bsdtar will not handle links well.
```

Comment:
This comment should explain the problem (symlinks can create a cyclic situation)